packages/dds/tree/src/simple-tree/core/unhydratedFlexTree.ts

```diff
@@ -121,26 +111,55 @@ export class UnhydratedFlexTreeNode implements FlexTreeNode {
 	 * Instead, it should always be acquired via {@link getOrCreateNodeFromInnerNode}.
 	 */
 	public constructor(
+		public readonly data: NodeData,
```

Comment:
Docs?

Reply:
Also, this constructor has existing `@param` docs that appear to be out of date.